src/core/common/HandleErrorModal/index.js

```diff
@@ -102,23 +104,38 @@ class HandleErrorModal extends Component {
         const errorStatus = get(httpErrors, 'status', null);
         const errorMessage = get(httpErrors, 'message', null);
 
+        let isWrongPatient = false;
+        if (errorMessage) {
```

Comment:
It's better to use a ternary condition in that case. That allows you to use a constant instead of a variable, which is simpler.

```js
const isWrongPatient = errorMessage ? errorMessage.includes('patient') : false;
```